README.md

```diff
@@ -121,53 +157,23 @@ export GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:~/catkin_ws/src/avoidance/sim/mode
 export QT_X11_NO_MITSHM=1
 
 # Setup some more Gazebo-related environment variables
-. ~/Firmware/Tools/setup_gazebo.bash ~/Firmware ~/Firmware/build/posix_sitl_default
+. ~/Firmware/Tools/setup_gazebo.bash ~/Firmware ~/Firmware/build/px4_sitl_default
 
 # Build and run simulation
-make posix_sitl_default gazebo
+make px4_sitl_default gazebo
 ```
 
-12. Add the Firmware directory to ROS_PACKAGE_PATH so that ROS can start the PX4.
+4. Add the Firmware directory to ROS_PACKAGE_PATH so that ROS can start the PX4.
 
 ```bash
 export ROS_PACKAGE_PATH=${ROS_PACKAGE_PATH}:~/Firmware
 ```
 
 You should now be ready to run the simulation.
```

Comment:
"You should now be ready to run the simulation." -> maybe "You should now be ready to run the simulation using local or global planner"